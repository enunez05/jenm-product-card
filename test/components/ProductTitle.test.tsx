import * as React from 'react';

describe('ProdcutTitle', () =>{
    test('Debe de mostrar el componente correctamente con el titulo', () => {})
})