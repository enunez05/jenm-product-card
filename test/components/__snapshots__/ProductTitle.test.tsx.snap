// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle Debe de mostrar el componente correctamente con el titulo 1`] = `
<span
  className="productDescription product-title undefined"
>
  custom product
</span>
`;

exports[`ProductTitle debe de mostrar el componente con el nombre del producto 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined undefined"
  >
    Coffe Mug - Sin Imagen
  </span>
</div>
`;
